bash/solutii/gheorghita.butnaru/compare.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Sa se scrie un fisier de comenzi, care verifica daca doua directoare sunt egale.
+# Numele celor doua directoare se transmit ca argumente in linia de comanda. Doua
+# directoare se considera ca sunt egale daca contin aceleasi subdirectoare si fisiere.
+# Se utilizeaza comanda diff.
+
```

Comment:
Nu ar trebui sa verifici si parametrii care sunt introdusi in script? In caz ca nu sunt corecti parametrii, se va afisa un mesaj de ajutor.